modin/data_management/functions/groupby_function.py

```diff
@@ -199,6 +195,14 @@ def try_filter_dict(agg_func, df):
         partition_dict = {k: v for k, v in agg_func.items() if k in df.columns}
         return lambda grp: grp.agg(partition_dict)
 
+    @staticmethod
+    def restore_default_function(agg_func, method=None, **kwargs):
+        if method == "size":
```

Comment:
Is there not a better way for handling this than using the string value? I prefer to keep things general enough and not special case if we can avoid it.

Reply:
Sometimes when we met unsupported case and defaulting to pandas, passed `map_func`, which was supposed to be executed in a MapReduce approach, won't work well. That's why we need some knowledge about the original operation to do fallback properly. 

In the last commit, I've got rid of the peace of code that you commented on and now explicitly pass the proper default function in case of `groupby_size`.